README.md

```diff
@@ -73,14 +73,15 @@ See the [Cassandra error handling done right blog](https://www.datastax.com/blog
 
 ## Useful links
 
-* [Manual](manual/)
+* [Manual]
 * [API docs]
 * Bug tracking: [JIRA]
 * [Mailing list]
 * [Changelog]
 * [FAQ]
 
-[API docs]: https://docs.datastax.com/en/drivers/java/4.17
+[Manual]: https://apache.github.io/cassandra-java-driver/4.19.0/
+[API docs]: https://apache.github.io/cassandra-java-driver/4.19.0/api/
```

Comment:
To be replaced by link for 4.19.2

Reply:
Also, in related news... we might want to (perhaps eventually) replace this with a link to "latest".  Static build tools usually don't support that idea but we can prolly make it work some other way.

But that's definitely a future goal.